### PR TITLE
Remove projects gallery

### PR DESCRIPTION
**Pull request by hennadialani**
Goodbye to projects gallery to make it cleaner + more aligned with what Stack's known for.

### CODE REVIEW

File: project-gallery.md [hunk 1]
```diff
@@ -1,11 +0,0 @@
----
-layout: projects
-title: Project Gallery
-permalink: /projects/
-stylesheets:
-  - /css/projects.css
```
Comment (nathunsmitty) [inline project-gallery.md]: you can also remove projects.css and projects.yml

Comment (nathunsmitty) [reply project-gallery.md]: tbh just grep for projects and see what comes up

Comment (hennadialani) [reply project-gallery.md]: should be good now!

**Comment by nathunsmitty** [comment]
swag

**Pull request merged**